docs/management/admin/admin-pg-ov.asciidoc

```diff
@@ -72,6 +72,13 @@ NOTE: Users must have permission to read/write to Fleet APIs to make changes to
 [role="screenshot"]
 image::images/integration-pg.png[Integration page]
 
+Users who have unique configuration and security requirements can select **Show Advanced Settings** to configure the Policy to support advanced use cases. Hover over each setting to view its description.
+
+Note: Advanced settings are not recommended for most users. 
```

Comment:
This needs to be in the "Note" CSS Style. So type `NOTE:` in the editor. After that's fixed you can merge this.